cmd/snap-seccomp/main_test.go

```diff
@@ -164,11 +165,25 @@ func (s *snapSeccompSuite) SetUpSuite(c *C) {
 	s.seccompSyscallRunner = filepath.Join(c.MkDir(), "seccomp_syscall_runner")
 	err = ioutil.WriteFile(s.seccompSyscallRunner+".c", seccompSyscallRunnerContent, 0644)
 	c.Assert(err, IsNil)
+
 	cmd = exec.Command("gcc", "-std=c99", "-Werror", "-Wall", "-static", s.seccompSyscallRunner+".c", "-o", s.seccompSyscallRunner, "-Wl,-static", "-static-libgcc")
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
 	err = cmd.Run()
 	c.Assert(err, IsNil)
+
+	// build again, this time syscall-runner -m32 multilib runner
+	cmd = exec.Command(cmd.Args[0], cmd.Args[1:]...)
+	cmd.Args = append(cmd.Args, "-m32")
+	for i, k := range cmd.Args {
+		if k == s.seccompSyscallRunner {
+			cmd.Args[i] = s.seccompSyscallRunner + ".m32"
+		}
+	}
+	err = cmd.Run()
+	if err != nil {
+		fmt.Printf("cannot build multi-lib syscall runner: %v", err)
+	}
```

Comment:
I wonder if it would be better to only build the m32 runner on the architectures that need it. As it is currently implemented, i386 will see 'cannot build multi-lib syscall runner: ...' in build output, which could be confusing. If you only compile on the architectures you care about testing (ie, amd64), then you could clean up your ppc64el code below.

Reply:
Yeah, I think that makes sense. I have done this now.